backend/app/debug_info.go

```diff
@@ -62,32 +64,47 @@ func addDefaultFactoryGameLog(writer *zip.Writer) error {
 	return nil
 }
 
+func redactFactoryGameLog(bytes []byte) []byte {
+	re := regexp.MustCompile(`(Added friend with nickname ').*(' on online context)`)
+	return re.ReplaceAll(bytes, []byte("${1}REDACTED${2}"))
+}
+
+func addLogFromPath(writer *zip.Writer, logPath string, zipFileName string) error {
+	bytes, err := os.ReadFile(logPath)
```

Comment:
`logPath` will not exist on the local disk if the install is a remote one, which is why I said before in discord that there's no way to merge the two game log functions.

`addLogFromPath` could take a reader or the file contents instead of the path (with an adjusted function name)